app/controllers/application_controller.rb

```diff
@@ -1,6 +1,15 @@
 class ApplicationController < ActionController::Base
   before_action :configure_permitted_parameters, if: :devise_controller?
 
+  # Redirect to a specific page on successful sign in and out
+  def after_sign_in_path_for(resource)
+    assets_path
```

Comment:
what if you make it your root_path?

Reply:
Root path changed to map_assests#index
Sign In/Out path changed to root_path in applicaiton_controller